pkg/cmd/infra/router/router.go

```diff
@@ -37,35 +54,75 @@ type templateRouterConfig struct {
 	StatsUsername      string
 }
 
+// f5RouterConfig is the config necessary to start an F5 router plugin
```

Comment:
That came from your skeleton at <https://github.com/pweil-/origin/commit/f057519d1ff91c4a34273c659e26148421ebe872>.  *grin*.  I'll fix it.